song_match/song_match.py

```diff
@@ -37,22 +39,73 @@ def __init__(self):
 
         self.__turn_on_cube_lights()
 
-        notes = [Note('E4'), Note('D4'), Note('C4')]
-        await self._robot.play_notes(notes)
+        # notes = [Note('E4'), Note('D4'), Note('C4')]
+        # await self._robot.play_notes(notes)
 
     async def __tap_handler(self, evt, obj=None, tap_count=None, **kwargs):
         cube = evt.obj
         note_cube = NoteCube(cube, self._song)
-        await note_cube.blink_and_play_note()
+        notes = self._song._notes
+        self._tappedNoteIndex = note_cube._cube.cube_id - 1
+        print("Tap Handler: " + str(self._tappedNoteIndex))
```

Comment:
You should remove debugging `print` statements like this.

Reply:
Yeah that was from when I was tracing the tap handler before I implemented the instance variable. Deleted the rest but forgot this one since it was by itself in a different spot.